pyproject.toml

```diff
@@ -34,12 +34,12 @@ bokeh = "^2.3.0"
 param = "^1.10.1"
 dill = "^0.3.3"
 pandas = "^1.2.3"
-numpy = "^1.20.1"
+numpy = "^1.19"
 jsonpickle = "^2.0.0"
 graphviz = "^0.16"
-networkx = "^2.5.1"
-matplotlib = "^3.4.1"
-h5py = "^3.3.0"
+networkx = "^2.6.0"
+matplotlib = "^3.5.2"
```

Comment:
just to make sure - matplotllib and h5py upgrade is a must?

Reply:
It is not. I reverted to older versions now.